### DISCUSSION TITLE
list index out of range.

### DISCUSSION THREAD

**Discussion by NameisNam** [volotat/SD-CN-Animation#75] [Q&A]
![image](https://github.com/volotat/SD-CN-Animation/assets/132860277/c955948e-1d47-4188-859c-d0438e70931f)

So I tried the first 2 times, it was working fine, but I interrupted it to try out newer prompt. 

Then I went to img2img to generate some images to work out the ideal prompt. 
Then I went back to vid2vid and try out the prompts now it doesn't work anymore.

![image](https://github.com/volotat/SD-CN-Animation/assets/132860277/cfa1c28e-1323-45f6-9272-864304d6e23f)

**Comment by jfranecki** [comment]
Try _Euler a_ as your sampling method.

**Comment by Apexius** [comment]
Not sure why this happens but it does so on occasion.. 


I can't guarantee this helps, but I found restoring to a previous backup has worked for me

in **_Extensions_** > **_Backup/Restore_** try to save a backup before the issue happens, if you come across the **IndexError** in the future you can just revert back and try again
